### PR TITLE
PMM-9015 Draft implementation to propose changes

### PR DESCRIPTION
**Pull request by percona-csalguero**
The only important change is the connectionRetry parameter.
The other changes were introduced by `gofumpt`

### CODE REVIEW

File: cmd/postgres_exporter/postgres_exporter.go [hunk 1]
```diff
@@ -990,11 +993,15 @@ func (s *Servers) GetServer(dsn string) (*Server, error) {
...
-	retries := 3
+
+	// Remove this retries and use the global connectionRetries parameter
+	// retries := 3
+
 	var server *Server
 	for {
-		if errCount++; errCount > retries {
-			return nil, err
+		// if errCount++; errCount > retries {
```
Comment (JiriCtvrtka) [inline cmd/postgres_exporter/postgres_exporter.go:1002]: I know that it is draft, but will be these lines which are commented out deleted after? Or is it for some purpose?

**Review by JiriCtvrtka — APPROVED** [review]

File: cmd/postgres_exporter/postgres_exporter.go [hunk 2]
```diff
@@ -990,10 +993,14 @@ func (s *Servers) GetServer(dsn string) (*Server, error) {
 	var err error
 	var ok bool
 	errCount := 0 // start at zero because we increment before doing work
-	retries := 3
+
+	// Remove this retries and use the global connectionRetries parameter
+	// retries := 3
```
Comment (ShashankSinha252) [inline cmd/postgres_exporter/postgres_exporter.go:998]: Why retain commented code?

**Review by ShashankSinha252 — CHANGES_REQUESTED** [review]

File: cmd/postgres_exporter/postgres_exporter.go (hunk 1 above)
Comment (percona-csalguero) [reply cmd/postgres_exporter/postgres_exporter.go:1002]: It was a draft PR to show where we needed a change. 
Now removed

File: cmd/postgres_exporter/postgres_exporter.go (hunk 2 above)
Comment (percona-csalguero) [reply cmd/postgres_exporter/postgres_exporter.go:998]: It was a draft PR to show where we needed a change. 
Now removed

**Pull request closed without merging**